.github/workflows/verify_install_arm64.yaml

```
@@ -0,0 +1,87 @@
+# © 2023 SolarWinds Worldwide, LLC. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at:http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+
+name: Verify Installation ARM64
+
+on:
+  workflow_dispatch:
+    inputs:
+      install-registry:
+        required: true
+        description: 'Registry used for install tests, e.g. pypi, testpypi, packagecloud'
+        type: choice
+        default: 'pypi'
+        options:
+        - pypi
+        - packagecloud
+        - testpypi
+      solarwinds-version:
+        required: false
+        description: 'Optional solarwinds-apm version, e.g. 0.0.3.2'
+
+env:
+  SOLARWINDS_APM_VERSION: ${{ github.event.inputs.solarwinds-version }}
+  SW_APM_COLLECTOR_AO_PROD: ${{ secrets.SW_APM_COLLECTOR_AO_PROD }}
+  SW_APM_COLLECTOR_PROD: ${{ secrets.SW_APM_COLLECTOR_PROD }}
+  SW_APM_COLLECTOR_STAGING: ${{ secrets.SW_APM_COLLECTOR_STAGING }}
+  SW_APM_SERVICE_KEY_AO_PROD: ${{ secrets.SW_APM_SERVICE_KEY_AO_PROD }}
+  SW_APM_SERVICE_KEY_PROD: ${{ secrets.SW_APM_SERVICE_KEY_PROD }}
+  SW_APM_SERVICE_KEY_STAGING: ${{ secrets.SW_APM_SERVICE_KEY_STAGING }}
+
+jobs:
+  launch-arm64:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.launch.outputs.matrix }}
+    steps:
+      - uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.CI_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.CI_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - id: launch
+        uses: solarwindscloud/ec2-runner-action@main
+        with:
+          action: launch
+          matrix: |
+            py3.7-ubuntu18.04
+            py3.11-ubuntu20.04
+          github-token: ${{ secrets.CI_GITHUB_TOKEN }}
+          runner-user: github
+          runner-directory: /github/actions
+          instance-type: t4g.medium
+          ami-name: gha-arm64-ubuntu22-.*
+          ami-owner: "858939916050"
+          subnet-id: subnet-0fd499f8a50e41807
+          security-group-ids: sg-0fd8d8cd6effda4a5
+
+  install-tests:
+    needs:
+      - launch-arm64
+    strategy:
+      matrix: |
+        py3.7-ubuntu18.04
+        py3.11-ubuntu20.04
```

The matrix syntax for the EC2 action and [the one for actual jobs](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs) are different since you can't have structured values as action inputs. This would have to look something like this to work, there's also a [VSCode extension for workflow syntax](https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-github-actions) that's really helpful for catching a lot of stuff in advance that is impossible to check locally.
```suggestion
      matrix:
        job:
          - py3.7-ubuntu18.04
          - py3.11-ubuntu20.04
```

---

Thanks @raphael-theriault-swi !! Changed in [b2575b2](https://github.com/solarwindscloud/solarwinds-apm-python/pull/154/commits/b2575b28c5cb18928681c25c38ea718435f9f8d4), and I've got that helpful extension installed now.